lib/rails_live_reload/rails/middleware/polling.rb

```diff
@@ -4,16 +4,16 @@ module Middleware
       class Polling < Base
         private
 
-        def rails_live_response(request)
+        def main_rails_live_response(request)
           [
             200,
             { 'Content-Type' => 'application/json' },
             [ RailsLiveReload::Command.new(request.params).payload.to_json ]
           ]
         end
 
-        def client_javascript
-          RailsLiveReload::Client.polling_js
+        def javascript_options
+          { polling_interval: RailsLiveReload.config.polling_interval }
```

Comment:
what if transform the keys to camelCase?
so the JS code can use `pollingInterval` here https://github.com/railsjazz/rails_live_reload/pull/14/files#diff-003f3a8b592a7c8a6682c527fb6de4bdb0465d61a3d62534161acb068efbdffcR23

Reply:
I think I would rather prefer to use ruby naming convention in such situations, after all it is a ruby gem.